### PR TITLE
Update ollama.py

### PR DESCRIPTION
**Pull request by gmgreg**
# Description

Enable base_url default value to be overwritten

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [ ] Added new notebook (that tests end-to-end)
- [x] I stared at the code and made sure it makes sense

# Suggested Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: llama_index/llms/ollama.py [hunk 1]
```diff
@@ -41,6 +39,7 @@ class Ollama(CustomLLM):
 
     def __init__(
         self,
+        base_url: str = "http://localhost:11434",
```
Comment (logan-markewich) [inline llama_index/llms/ollama.py:42]: You'll need to pass this into the init below

Comment (gmgreg) [reply llama_index/llms/ollama.py:42]: Thought I did (line 42), not correct?

Comment (gmgreg) [reply llama_index/llms/ollama.py]: Ok, moved it to 44 to pass linting.

Comment (logan-markewich) [reply llama_index/llms/ollama.py]: Sorry, I meant the super init, around line 56

Comment (logan-markewich) [reply llama_index/llms/ollama.py]: added it

**Pull request merged**